src/components/UI/GlobalStyles/GlobalStyles.js

```diff
@@ -9,18 +9,26 @@ const GlobalStyle = createGlobalStyle`
   html {
   font-size: 62.5%;
   font-family: Roboto Mono, sans-serif;
+  overflow: hidden;
+  overflow-y:auto; 
 }
+
 body {
-  background-color: black;
-  height: auto;
-  color: white;
+  padding: 0;
+	margin: 0;
+  background-color: black; 
+  
+  
+  
 }
 
+
 :root {
 --gradient: rgba(59, 173, 227, 1) 0%,
   rgba(87, 111, 230, 1) 25%,
```

Comment:
I'm not really sure what all these lines are supposed to do. Maybe I'm wrong, where/how do you use this? I thought you were supposed to assign the colors to a variable if you want to use them in other places in your app?

Reply:
Hi Cintia,

thanks for taking your time to look over my code. As for your remarks I applied all (thanks again for seeing that, there were so many things I overlooked) but the one about the :root as that was told by a coach as those are gradient colors.